cli/integration_tests/composable_config/composing-add-keys.t

```diff
@@ -0,0 +1,100 @@
+Setup
+  $ . ${TESTDIR}/../setup.sh
+  $ . ${TESTDIR}/setup.sh $(pwd) ./monorepo
+
+# The add-keys-task in the root turbo.json has no config. This test:
+# [x] Tests dependsOn works by asserting that another task first
+# [x] Tests outputs works by asserting that the right directory is cached
+# [x] Tests outputMode by asserting output logs on a second run
+# [x] Tests inputs works by changing a file and testing there was a cache miss
+# [x] Tests env works by setting an env var and asserting there was a cache miss
+
+# 1. First run, assert for `dependsOn` and `outputs` keys
+  $ ${TURBO} run add-keys-task --skip-infer --filter=add-keys > tmp.log
```

Comment:
Would like to debug why `--skip-infer` is needed here with you. On my machine it works fine without which probably indicates we're letting some host related configs affect these tests 😢

Reply:
Tried removing and tests still pass. It's possible it was user error or something was changed/fixed after I started working on these tests. In any case, all's well now!